### PR TITLE
Trivy recommended hardening

### PR DESCRIPTION
**Pull request by jcpunk**
These hardening bits were recommended by the trivy scanner.

https://github.com/aquasecurity/trivy
https://github.com/aquasecurity/trivy-operator

### CODE REVIEW

**Comment by derekbit** [comment]
@jcpunk 
Thanks for your contribution.
Could you please link me to a reference for the feature? Thank you.

**Comment by derekbit** [comment]
BTW, could you help resolve the conflict? Thank you.

**Comment by derekbit** [comment]
Not sure if the hardening can introduce side effect?
I think we can gray the block out and leave a comment here. Users can enable it by updating the values.yaml.
@jcpunk WDYT?

**Comment by jcpunk** [comment]
I'd still like to see some version of this.

**Comment by derekbit** [comment]
Let's make it in v0.0.28.

**Comment by jcpunk** [comment]
I'd still like to see some version of this.

File: deploy/chart/local-path-provisioner/values.yaml [hunk 1]
```diff
@@ -74,10 +74,10 @@ podAnnotations: {}
 podSecurityContext: {}
   # runAsNonRoot: true
 
-securityContext: {}
+securityContext:
+  seccompProfile:
+    type: RuntimeDefault
```
Comment (derekbit) [inline deploy/chart/local-path-provisioner/values.yaml:79]: I'm wondering if we need to enable it by deafult. Or, users can enable it if they need it.

Comment (jcpunk) [reply deploy/chart/local-path-provisioner/values.yaml:79]: I would aim for by default and have folks loosen the security restrictions if they can't run with them.

**Comment by jcpunk** [comment]
conflicts resolved

**Comment by jcpunk** [comment]
I'd still like to see some version of this.

**Pull request closed without merging**